charts/linkerd-crds/values.yaml

```diff
@@ -1,3 +1,4 @@
-enableHttpRoutes: true
-enableTlsRoutes: true
-enableTcpRoutes: true
+enableHttpRoutes: false
+enableTlsRoutes: false
+enableTcpRoutes: false
+installGatewayAPI: true
```

Comment:
Probably worth comments explaining the relationship? And I'd put `installGatewayAPI` first, since it's the most important.